README.md

```diff
@@ -17,35 +14,41 @@ This SDK is built on the AWS Common Runtime, a collection of libraries
 cross-platform, high-performance, secure, and reliable. The libraries are bound
 to Java by the [aws-crt-java](https://github.com/awslabs/aws-crt-java) package.
 
-Integration with AWS IoT Services such as
-[Device Shadow](https://docs.aws.amazon.com/iot/latest/developerguide/iot-device-shadows.html)
-[Jobs](https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html)
-[Fleet Provisioning](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html)
-is provided by code that been generated from a model of the service.
 
-# Installation
-## Minimum Requirements
+*__Jump To:__*
+* [Installation](#Installation)
+* [Samples](samples)
+* [Getting Help](#Getting-Help)
+* [Giving Feedback and Contributions](#Giving-Feedback-and-Contributions)
+* [More Resources](#More-Resources)
+
+
+
+## Installation
+
+### Minimum Requirements
 *   Java 8 or above
 
 ** Set JAVA_HOME first
 
 *   Install Maven
-
+```
 brew install maven
+```
 
-## Requirements to build the AWS CRT locally
+### Requirements to build the AWS CRT locally
 *   CMake 3.1+
 *   Clang 3.9+ or GCC 4.4+ or MSVC 2015+
 
-## Build IoT Device SDK from source
+### Build IoT Device SDK from source
 ```
 git clone https://github.com/awslabs/aws-iot-device-sdk-java-v2.git
 # update the version of the CRT being used
 mvn versions:use-latest-versions -Dincludes="software.amazon.awssdk.crt*"
 mvn clean install
 ```
 
-## Build CRT from source
+### Build CRT from source
```

Comment:
I'd like Justin's feedback but I think this should be removed in favor of a link to the CRT readme.